families/families.js

```diff
@@ -8,35 +9,69 @@ const logoutButton = document.getElementById('logout');
 logoutButton.addEventListener('click', () => {
     logout();
 });
+// const familiesList = document.getElementById('families-list');
 
-function displayFamilies() {
+async function displayFamilies() {
+    familiesEl.textContent = '';
     // fetch families from supabase
-
+    const families = await getFamilies();
+    console.log(families);
     // clear out the familiesEl
 
     for (let family of families) {
-        // create three elements for each family, one for the whole family, one to hold the name, and one to hold the bunnies
-        // your HTML Element should look like this:
-        // <div class="family">
-        //    <h3>the Garcia family</h3>
-        //    <div class="bunnies">
-        //        <div class="bunny">Fluffy</div>
-        //        <div class="bunny">Bob</div>
-        //    </div>
-        // </div>
-        // add the bunnies css class to the bunnies el, and family css class to the family el
-        // put the family name in the name element
-        // for each of this family's bunnies
-        //    make an element with the css class 'bunny', and put the bunny's name in the text content
-        //    add an event listener to the bunny el. On click, delete the bunny, then refetch and redisplay all families.
-        // append this bunnyEl to the bunniesEl
-    }
+        // familiesEl.append(renderFamily(family));
+        const familyEl = document.createElement('div');
+        const nameEl = document.createElement('p');
+        const bunniesEl = document.createElement('div');
+
+        bunniesEl.classList.add('bunnies');
+        familyEl.classList.add('family');
+
+        nameEl.textContent = family.name;
+
+        for (let bunny of family.fuzzy_bunnies) {
+            const bunnyEl = renderBunny(bunny);
+
+            bunnyEl.classList.add('bunny');
+            bunnyEl.textContent = bunny.name;
 
-    // append the bunniesEl and nameEl to the familyEl
+            bunnyEl.addEventListener('click', async () => {
+                await deleteBunny(bunny.id);
 
-    // append the familyEl to the familiesEl
+                const newFamilies = await getFamilies();
+
+                displayFamilies(newFamilies);
+            });
```

Comment:
Here we call displayFamilies with a parameter of newFamilies, however, our actual function doesn't take any parameters. Usually this would cause a console error, just be aware of it in the future :)